lib/Resizable.js

```diff
@@ -182,6 +186,27 @@ export default class Resizable extends React.Component<Props, State> {
       const canDragX = (this.props.axis === 'both' || this.props.axis === 'x') && ['n', 's'].indexOf(axis) === -1;
       const canDragY = (this.props.axis === 'both' || this.props.axis === 'y') && ['e', 'w'].indexOf(axis) === -1;
 
+      // Check if handle's position has moved since last resizeHandler call - adjust deltas accordingly
+      if (this.draggingNode == null && e.target instanceof HTMLElement) {
```

Comment:
Is saving this ref necessary (rather than simply using `e.target`)?

Reply:
I originally had it just using `e.target` on each callback but because the listener is calling back to any movement in the document body, `e.target` will not always be the same value during a drag. It will be whatever element the mouse is currently over. As you can see, using this value directly causes jumpy behaviour when the mouse enters different elements:
[![Image from Gyazo](https://i.gyazo.com/977571a2e3aa46db3d1702a94ee38a27.gif)](https://gyazo.com/977571a2e3aa46db3d1702a94ee38a27),

Hence the need to persist a reference to the same element through multiple callbacks

Reply:
Or an even worse demo:
[![Image from Gyazo](https://i.gyazo.com/14e62780c79298e17990cb785faf0bd2.gif)](https://gyazo.com/14e62780c79298e17990cb785faf0bd2)